data/scenarios/02Tutorials/06-scan.yaml

```diff
@@ -0,0 +1,62 @@
+name: Scan
+description: |
+  Learn how to send robots to safely scan your surroundings.
+entities:
+  - name: Goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - When you land on a alien planet, all the entites in the
+      world will be unfamiliar to you.
+    - But you can learn what they are using the scanner device.
+    - Send robots to move to the unknown entites (marked as '?'),
+     'scan forward' and after that come back and 'upload base'.
```

Comment:
We should probably explain `scan` a bit better.  I note your solution uses `scan left`.

Reply:
I expanded the description a little, but feel free to make a `suggestion` with better wording. :wink: